bundles/org.openhab.ui/web/src/assets/i18n/theme-switcher/de.json

```diff
@@ -0,0 +1,15 @@
+{
+  "about.theme": "Erscheinungsbild",
+  "about.theme.auto": "Auto",
```

Comment:
Automatisch

Reply:
Doesn't work, it's for the little square box, I updated to "Auto." with a punkt to signify it's an abbreviation.
You'll be able to debate about this on Crowdin later ;)

Reply:
Okay, I see, not that big thing. Let me know when Crowdin is on start. I already support makecode for micro:bot ;)